incubator/hnc/test/e2e/issues_test.go

```diff
@@ -20,19 +20,19 @@ var _ = Describe("Issues", func() {
 	)
 
 	BeforeEach(func() {
-		CleanupNamespaces(nsParent, nsChild, nsSub1, nsSub2, nsSub1Sub1, nsSub2Sub1,
-			nsSubSub2, nsSubChild, nsSubSubChild)
+		CleanupTestNamespaces()
 	})
 
 	AfterEach(func() {
-		CleanupNamespaces(nsParent, nsChild, nsSub1, nsSub2, nsSub1Sub1, nsSub2Sub1,
-			nsSubSub2, nsSubChild, nsSubSubChild)
+		CleanupTestNamespaces()
 	})
 
 	It("Should not delete full namespace when a faulty anchor is deleted - issue #1149", func() {
 		// Setup
-		MustRun("kubectl create ns", nsParent)
-		MustRun("kubectl hns create", nsChild, "-n", nsParent)
+		CreateNamespace(nsParent)
+
+		CreateSubnamespace(nsChild,nsParent)
+
```

Comment:
I'd skip the extra newlines?

Reply:
Sorry about the new lines, my search replace of the old code resulted in these.Will fix.

Reply:
np!